core/sequencer/sequencer_api.proto

```diff
@@ -56,6 +56,8 @@ message RunBatchRequest {
   int32 min_batch = 2;
   // max_batch is the maximum number of items in a batch.
   int32 max_batch = 3;
+  // block until a Signed Log Root has been published which encompases all map roots.
```

Comment:
Is it all map roots, or just one map root (that contains all mutations of the batch)?